### PR TITLE
EWR code-samples added

### PR DESCRIPTION
**Pull request by bilalyasar**

### CODE REVIEW

File: enterprise/wan-replication/src/main/java/Readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<h2>ABOUT</h2>
+This project is prepared to serve as a sample application for Hazelcast Enterprise
+Here, Hazelcast's use case is Enterprise Wan Replication. 
+
+ 
+<h3>Scenario</h3>
+
+Very briefly, you can send map or cache replication events via Wan Replication.
+In this project we have two cluster ( clusterA and clusterB) and clusterA replicates events to
```
Comment (emrahkocaman) [inline enterprise/wan-replication/src/main/java/Readme.md]: should be "clusters" :)

File: enterprise/wan-replication/src/main/java/Readme.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+clusterB. Also there is a command line interface that you can put objects from clusterA and check the objects in clusterB.
+
+In this project there are two sample application. One of them is for Map Replication and the other one is for Cache Replication.
```
Comment (emrahkocaman) [inline enterprise/wan-replication/src/main/java/Readme.md]: Same here :) "applications"

**Comment by bilalyasar** [comment]
@emrahkocaman thanks for reviewing, updated.

**Comment by emrahkocaman** [comment]
Bilal, can you please modify the sample that it also shows [BatchWanReplication](https://github.com/hazelcast/hazelcast-enterprise/blob/master/hazelcast-enterprise/src/main/java/com/hazelcast/enterprise/wan/replication/WanBatchReplication.java) usage with and without `snapshot-enabled`functionality.

It would be nice if sample app takes replication implementation as a command line startup parameter maybe.

**Pull request merged**